org.civicrm.styleguide/styleguide.php

```diff
@@ -123,3 +123,41 @@ function styleguide_civicrm_angularModules(&$angularModules) {
 function styleguide_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
   _styleguide_civix_civicrm_alterSettingsFolders($metaDataFolders);
 }
+
+/**
+ * Implements hook_civicrm_navigationMenu().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_navigationMenu
+ */
+function styleguide_civicrm_navigationMenu(&$menu) {
+  _styleguide_civix_insert_navigation_menu($menu, 'Support/Developer', array(
+    'label' => ts('Style Guide', array('domain' => 'org.civicrm.styleguide')),
+    'name' => 'developer_styleguide',
+    'permission' => 'access CiviCRM',
+    'operator' => 'OR',
+    'separator' => 0,
+  ));
+  foreach (Civi::service('styleguides')->getAll() as $name => $defn) {
+    _styleguide_civix_insert_navigation_menu($menu, 'Support/Developer/developer_styleguide', array(
+      'label' => $defn['label'],
+      'name' => 'developer_styleguide_' . $defn['name'],
+      'url' => 'civicrm/styleguide/' . $defn['name'],
```

Comment:
oh, I see the name is used to build the url. So maybe it's good to add some parsing on the CRM_Styleguide_Manager in order to avoid invalid URL chars